docs/integrations/containers-orchestration/opentelemetry/rabbitmq-opentelemetry.md

```diff
@@ -0,0 +1,146 @@
+---
+id: rabbitmq-opentelemetry
+title: RabbitMQ - OpenTelemetry Collector
+sidebar_label: RabbitMQ - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for RabbitMQ.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/rabbitmq.png')} alt="icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [RabbitMQ](https://www.rabbitmq.com/getstarted.html) app is a unified log app. Preconfigured dashboards provide insight into error logs. RabbitMQ logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/RabbitMq-OpenTelemetry/RabbitMQ-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for RabbitMQ
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of RabbitMQ App install if not already present.
+
+**messaging.cluster.name** - User configured. Specify the user-friendly cluster name which RabbitMQ belongs to.
```

Comment:
The field listed here are getting rendered as a paragraph and not two seperate points.